machine-learning/app/config.py

```diff
@@ -39,6 +39,10 @@ class Config:
         case_sensitive = False
         env_nested_delimiter = "__"
 
+    @property
+    def device_id(self) -> int:
+        return os.environ.get("MACHINE_LEARNING_DEVICE_ID", "0")
```

Comment:
The reason this is a property is because the environmental variables are normally evaluated before the pre-fork hook runs. Using a property makes it evaluate the post-fork value instead.